gruntfile.js

```diff
@@ -204,6 +209,33 @@ function shell_download_docker_binary(p, a) {
   }
 }
 
+function shell_download_docker_compose_binary(p, a) {
+  var ps = { windows: 'win', darwin: 'mac' };
```

Comment:
I know other functions use this `ip` and `ia` syntax, but can we give better names so we understand the meaning (i.e p -> `platform`, a -> `arch`, etc.)

Reply:
good point, we should harmonize the scripts and make them more explicit